.circleci/config.yml

```diff
@@ -0,0 +1,72 @@
+version: 2.1
+
+commands:
+  before_install:
+    steps:
+      - run: |
+          dnf install make -y
+
+  install:
+    steps:
+      - run:
+          name : list_folder
+          command : ls
+      - run:
+          name : install_cmd
+          command: make install
+  
+  before_script:
+    steps:
+      - run: |
+          sudo -u nuodb "/opt/nuodb/etc/nuoadmin" tls $NUO_SET_TLS
```

Comment:
Curious where does the NUO_SET_TLS variable come from?

Reply:
See line 64.